README.md

```diff
@@ -105,12 +107,20 @@ micromamba activate xeus-lite-host
 python -m pip install jupyterlite-xeus
 jupyter lite build --XeusAddon.prefix=$PREFIX
 ```
+
+We now need to shift necessary files like `xcpp.data` which contains the binary representation of the file(s)
+we want to include in our application. As of now this would contain all important files like Standard Headers,
+Libraries etc coming out of emscripten's sysroot. Assuming we are still inside build we should do the following
+```bash
+cp xcpp.data _output/extensions/@jupyterlite/xeus/static
+cp $PREFIX/lib/libclangCppInterOp.so _output/extensions/@jupyterlite/xeus/static
```

Comment:
Though we would always need to copy `xcpp.data` to a location that is accessible by Jupyter Lite (either `_output/xeus/bin` or `_output/extensions/@jupyterlite/xeus/static` that already have access to xcpp.js and xcpp.wasm .......

The second line here shifting `libclangCppInterOp.so` would only be required temporarily. This is only because JupyterLite doesn't deal with shared libs as expected as I think it expects static libs. Hence once we shift to emsdk 3.1.73 and can support a static lib (libclangCppInterOp.a) this would no longer be required !

Hence this is temporary.